Source/Shaders/PostProcessFilters/AmbientOcclusion.glsl

```diff
@@ -0,0 +1,11 @@
+uniform sampler2D colorTexture;
```

Comment:
Should this file have a more precise name?  Like `AmbientOcclusionModulate.glsl`.